numba/tests/test_return_values.py

```diff
@@ -31,13 +30,12 @@ class TestReturnValues(unittest.TestCase):
     def test_nopython_func(self, flags=enable_pyobj_flags):
         # Test returning func that is supported in nopython mode
         pyfunc = get_nopython_func
-        cr = compile_isolated(pyfunc, (), flags=flags)
-        cfunc = cr.entry_point
+        cfunc = jit((), **flags)(pyfunc)
         if flags == enable_pyobj_flags:
             result = cfunc()
             self.assertEqual(result, abs)
         else:
-            result = cfunc()
+            cfunc()
```

Comment:
Dead code because the compiler would have failed.

Reply:
```suggestion
            self.fail("unexpected to compile successfuly")
```

Reply:
Thanks, fixed in 90ffc0544d1.